token/services/ttxdb/db/sql/random.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sql
+
+import (
+	"crypto/rand"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	// NonceSize is the default NonceSize
+	NonceSize = 24
+)
+
+// GetRandomBytes returns len random looking bytes
+func GetRandomBytes(len int) ([]byte, error) {
```

Comment:
In other places, the SQL driver uses go-uuid to generate a unique ID (see https://github.com/hyperledger-labs/fabric-token-sdk/blob/ad5e98882ce2cad448e1e4f397f912e416203552/token/services/ttxdb/db/sql/sql.go#L177). I propose to stick to that.

Reply:
Perfect, I'll use that.